DeprecationWarning: The `hash` argument is deprecated in favor of `unsafe_hash`

When running some flake8 tests also including flake8-bugbear I can see the following DeprecationWarning

```
.nox/test_flake8-3-10-flake8-7/lib/python3.10/site-packages/bugbear.py:49: in <module>
    @attr.s(hash=False)
.nox/test_flake8-3-10-flake8-7/lib/python3.10/site-packages/attr/_make.py:1291: in attrs
    warnings.warn(
E   DeprecationWarning: The `hash` argument is deprecated in favor of `unsafe_hash` and will be removed in or after August 2025.
```

--- a/bugbear.py
+++ b/bugbear.py
@@ -46,7 +46,7 @@ B902_default_decorators = {"classmethod"}
 Context = namedtuple("Context", ["node", "stack"])
 
 
-@attr.s(hash=False)
+@attr.s(unsafe_hash=False)
 class BugBearChecker:
     name = "flake8-bugbear"
     version = __version__

